### DISCUSSION TITLE
install-globaleaks.sh not work.

### DISCUSSION THREAD

**Discussion by vdlsupport** [globaleaks/globaleaks-whistleblowing-software#3543] [Q&A]
### What version of GlobaLeaks are you using?

4.11.5

### What operating system(s) are you seeing the problem on?

Linux

### Describe the issue

On system with OS Debian-bullseye, after start install-globaleaks.sh the script stop with the following error.

...
Updating GlobaLeaks apt source.list in /etc/apt/sources.list.d/globaleaks.list ...
Running: "apt-get update -y"... SUCCESS
Running: "apt-get install globaleaks -y"... FAIL
Ouch! The installation failed.
COMBINED STDOUT/STDERR OUTPUT OF FAILED COMMAND:
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package globaleaks

The contents of file source is correct:
# cat /etc/apt/sources.list.d/globaleaks.list
deb http://deb.globaleaks.org bullseye/

Can someone help me to solve this issue?
Thanks

**Comment by evilaliv3** [comment]
Thank you for your question.

You should have an install.log file next to the install script; would you please share it?

**Comment by vdlsupport** [comment]
Install.log has very poor information.

root@cnb394-rhv:/var/tmp# cat install.log
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package globaleaks

**Comment by evilaliv3** [comment]
I see. We are sorry for this. Please let us know in case you identify a solution for your situation.

Thank you!
